### DISCUSSION TITLE
Adding missing lithium plating parameters to a parameter set

### DISCUSSION THREAD

**Discussion by aminacoric** [pybamm-team/PyBaMM#2009] [Q&A] [answered]
Hi, I would like to run a simulation for a lithium-ion battery model using the Marquis2019 parameter set including lithium plating (LP) degradation. I noticed that there are some parameters missing to be able to run the simulation with the LP-option added. I have tried to add the missing LP parameters (okane2020_Li_plating or yang2017_Li_plating) to the Marquis2019 parameter set and the simulation is executable. But I noticed that only the parameter sets for NMC batteries had the LP parameters already included. Can I just add the LP parameters to the Marquis2019 parameter set even though it is a LCO battery? Are the available LP parameters reasonable for the LCO battery?

**Comment by brosaplanella** [comment, marked as answer]
It is hard to say if the parameters would be applicable to your chemistry, might be worth just trying if they give you reasonable results. To add plating parameters to an existing one you can do it via defining a new chemistry (see [this notebook for an example on how to manage chemistries](https://github.com/pybamm-team/workshop-notebooks/blob/main/2021-09-workshop/Session%203/1-parameter-management.ipynb)).
